src/components/Title.js

```diff
@@ -0,0 +1,5 @@
+import style from "./Title.module.scss";
+
+export const Title = () => {
+  return <div className={style.titleStyle}>Everything is possible</div>;
```

Comment:
```suggestion
  return <div className={style.title}>Everything is possible</div>;
```

it is obvious it is a style, no need to mention it in the name

Reply:
understood, fixed